sync2jira/upstream_issue.py

```diff
@@ -181,6 +181,126 @@ def handle_github_message(body, config, is_pr=False):
     return i.Issue.from_github(upstream, issue, config)
 
 
+def handle_pagure_message(body, config):
+    """
+    Handle Pagure message from FedMsg.
+
+    :param Dict body: FedMsg Message
+    :param Dict config: Config File
+    :returns: Issue object
+    :rtype: sync2jira.intermediary.Issue
+    """
+    upstream = body["project"]["name"]
+    ns = body["project"].get("namespace") or None
+    if ns:
+        upstream = "{ns}/{upstream}".format(ns=ns, upstream=upstream)
```

Comment:
The `or None` is pointless:  if the `get()` "fails", it will return `None` anyway (which, will evaluate to `False` and result in the RHS of the `or` being returned which will result in `None`...).

(It's true that the `get()` could return some _other_ False-y value which would then be replaced by `None`, but the next line is going to handle _all_ False-y values, so there is no need to make the replacement.)